kubernetes/service.yaml

```diff
@@ -36,8 +36,10 @@ spec:
       #   port: 9180
       #   protocol: TCP
       #   targetPort: 9180
-      selector:
-        app: apisix-gw
-      type: NodePort
-      externalTrafficPolicy: Local
-      # sessionAffinity: None
+  selector:
+    app: apisix-gw
+  type: NodePort
+  externalTrafficPolicy: Local
+  # sessionAffinity: None
+  externalIPs:
+    - 10.253.0.11
```

Comment:
Why chose this IP?

Reply:
I could not find method to randomly choose external IP, so I just randomly choose one. Is there any accustomed choice?

Reply:
Well, Do we really need the external IP?

Reply:
Why not use the NodePort service?

Reply:
Thanks for the suggestions! Changed to port forward